apps/judicial-system/web/src/components/SigningModal/SigningModal.tsx

```diff
@@ -23,8 +23,8 @@ import { RulingSignatureConfirmationQuery } from '../../utils/mutations'
 import { Modal } from '..'
 import MarkdownWrapper from '../MarkdownWrapper/MarkdownWrapper'
 import {
-  RequestRulingSignatureMutationMutation,
-  RulingSignatureConfirmationQueryQuery,
+  RequestRulingSignatureMutation as RequestRulingSignatureMutationMutation,
```

Comment:
I'm leaving these with the MutationMutation name because this is going to be the first place I'm going to integrate the change using the hook from the schema in my next PR